osquery/config/parsers/query_packs.cpp

```diff
@@ -51,7 +51,7 @@ bool versionChecker(const std::string& pack, const std::string& version) {
 bool platformChecker(const std::string& required, const std::string& platform) {
   // Match if platform is 'ubuntu12' and required is 'ubuntu'.
   // Do not match if platform is 'ubuntu12' and required is 'ubuntu14'.
-  return (platform.find(required) == std::string::npos);
+  return (platform.find(required) != std::string::npos);
```

Comment:
Do you have an example? I'd like to support: linux2, linux3, or freebsd10, darwin14, etc